src/test/java/bio/terra/stairway/FlightMapTest.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.stairway;
+
+import bio.terra.stairway.fixtures.FlightsTestNonPojo;
+import bio.terra.stairway.fixtures.FlightsTestPojo;
+import java.util.UUID;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Tag("unit")
+public class FlightMapTest {
+
+  private static final Logger logger = LoggerFactory.getLogger("FlightMapTest");
+
+  @Test
+  public void toAndFromJson() {
+    FlightMap outMap = new FlightMap();
+
+    FlightsTestPojo pojoIn = new FlightsTestPojo().anint(99).astring("mystring");
+    outMap.put("mypojo", pojoIn);
+
+    FlightsTestNonPojo nonPojoIn = new FlightsTestNonPojo(1.25f);
+    outMap.put("mynonpojo", nonPojoIn, FlightsTestNonPojo.serializer());
+
+    Integer intIn = 3;
+    outMap.put("myint", intIn);
+
+    String json = outMap.toJson();
+    logger.info("JSON: '{}'", json);
```

Comment:
consider removing the logger.info's in tests. If there's particularly helpful context for debugging an assertion, add it as a message to the `assertEquals`.

Reply:
Main intent was to opportunistically generate test vectors for string literals in test cases.  OK to move this to DEBUG log level, or is the preference to remove?